packages/python/plotly/js/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "scripts": {
+    "build": "esbuild --bundle --alias:plotly.js=plotly.js/dist/plotly --format=esm --minify --outfile=../plotly/bundle.js widget.ts",
```

Comment:
I added the `--minify` and `--alias` flags. `minify` makes the final bundle smaller. 

The alias here is to make the import nicer for typescript type checking types (tsc), while esbuild needs to point to the actual files (./dist).